content/tutorial/01-simple-orders/01-order/03-sign-order/README.md

```diff
@@ -20,12 +20,18 @@ For signing, we will use the `UnsignedOrder` type from the `SDK`, along with the
 /// file: run.ts
 import type { Web3Provider } from '@ethersproject/providers';
 +++import { OrderBookApi, SupportedChainId, OrderQuoteRequest, OrderQuoteSideKindSell, OrderSigningUtils, UnsignedOrder } from '@cowprotocol/cow-sdk';+++
++++import { BigNumber } from 'ethers';+++
```

Comment:
what are all these ++++?

Reply:
Piggybacking on the existing code, it adds a highlight to the row:

![image](https://github.com/user-attachments/assets/d1f34c24-e97c-4f19-9be0-a8e408189e77)

Reply:
Anyway, no longer using `BigNumber`, removed the import.